### PR TITLE
Update whitelist

### PR DESCRIPTION
**Pull request by nathanielcwm**

### CODE REVIEW

File: _data/whitelist.yml [hunk 1]
```diff
@@ -1,198 +1,240 @@
+---
 projects:
-- name: "Amicable Numbers"
-  link: "https://sech.me/boinc/Amicable/"
```
Comment (barton2526) [inline _data/whitelist.yml:3]: why are the quotes being removed?

Comment (nathanielcwm) [reply _data/whitelist.yml:3]: I just copied it directly from Gridcoin-Site

Comment (barton2526) [reply _data/whitelist.yml:3]: This is not the same as the .us website.  Please follow the [readme](https://github.com/jring-o/jring-o.github.io/blob/master/_data/readme.md).

Comment (nathanielcwm) [reply _data/whitelist.yml:3]: Removal of quotes seem to have originated from b5b20d8

Comment (nathanielcwm) [reply _data/whitelist.yml:3]: > This is not the same as the .us website.  Please follow the [readme](https://github.com/jring-o/jring-o.github.io/blob/master/_data/readme.md).

looks to be the same with minor differences e.g: it details how to write blog posts here and doesn't have an exchange template or gdpr specific steps

Comment (nathanielcwm) [reply _data/whitelist.yml:3]: pythonix's previous commits to the whitelist template retained the gdpr status / steps and I don't see why they should be removed

Comment (nathanielcwm) [reply _data/whitelist.yml:3]: @roboticmind why did you remove the quotes in b5b20d8?

Comment (barton2526) [reply _data/whitelist.yml:3]: hmm, maybe it's fine then.  Still need the CI to be approved

Comment (RoboticMind) [reply _data/whitelist.yml:3]: The quotes were probably removed by some script I used to sort it more easily. I put something together quickly that went from YAML -> JSON (and sorted it there) -> YAML and the last conversion probably removed the quotes. It makes no difference in this case, but quotes are better in general

**Pull request merged**